### PR TITLE
Replace bintray

### PR DESCRIPTION
**Pull request by menglewis**
Bintray is being deprecated which will cause builds to fail because there are dependencies on Bintray: https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

Also see https://github.com/graphframes/graphframes/issues/384

This PR replaces the usages of Bintray with `repo.scala-sbt.org` which I found from this [issue on SBT](https://github.com/sbt/sbt/issues/6294) about replacing Bintray. I was able to run a build locally with these changes during the April 26 Bintray brownout.

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // You may use this file to add plugin dependencies for sbt.
-resolvers += "Spark Packages repo" at "https://dl.bintray.com/spark-packages/maven/"
+resolvers += "Spark Packages repo" at "https://repo.scala-sbt.org/scalasbt/sbt-plugin-releases/"
```
Comment (bozhang2820) [inline project/plugins.sbt:2]: This should be changed to `https://repos.spark-packages.org/` instead. See: http://apache-spark-user-list.1001560.n3.nabble.com/Bintray-replacement-for-spark-packages-org-td40128.html

Comment (menglewis) [reply project/plugins.sbt]: I'm not sure if I have permissions to re-request a review, but I've made this change and this PR is ready for review again.

**Comment by Inkvi** [comment]
Can this PR be reviewed and merged? With bintray down, it's impossible to include package from the maven repository.

**Comment by pvillacorta-stratio** [comment]
The issue is still there for instance if you need to import it from cloud platforms. E.g. in Google Cloud Dataproc, AFAIK you cannot specify the locaton of the dependencies as it always goes to Maven, therefore it fails to create the cluster with the GraphFrames dependency. 

I really hope the PR can be merged soon :-)

**Comment by menglewis** [comment]
Is there anything else I need to do to get this merged? Happy to help if there's any way I can help.

**Comment by mengxr** [comment]
Merged into master. Thanks!!

**Pull request merged**